### PR TITLE
Update advisor.yml

### PR DESCRIPTION
**Pull request by kuklas**
Adding cards on the top from Advisor for 

- Pathways, 
- Incidents, 
- Critical recommendations, 
- Important recommendations.

### CODE REVIEW

**Comment by kuklas** [comment]
@mmenestr does this make sense pls?

File: data/advisor.yml [hunk 1]
```diff
@@ -3,6 +3,18 @@ advisor:
   color: 2
   features:
     _scope: '[data-ouia-app-id="advisor"]'
+    "Advisor - Card: Pathways": 
+      selectors:
+        - '{}[data-testid="Pathways"]'
```
Comment (mmenestr) [inline data/advisor.yml:8]: I think you need a space between the {} and the rest of the selector. Otherwise looks good.
Same for all of them

Comment (kuklas) [reply data/advisor.yml:8]: but the rest of the existing ones do not have that space. Or am I misunderstanding the position of the space

Comment (ryelo) [reply data/advisor.yml:8]: They should all have the space

https://github.com/RedHatInsights/pendo-config-automation/blob/main/docs/data.md#scope

**Comment by ryelo** [comment]
Hey, sorry -- On my review, I was just talking about the ones that you added, not the rest of the file. 

Everything that was there already is spaced correctly. It's just due to how the selectors appear in the DOM, the ones you added need that extra space.

**Comment by kuklas** [comment]
we are getting there @ryelo  hopefully this is the last edit 🔪

**Review by justinorringer — APPROVED** [review]
Looks good now!

**Pull request merged**